app/controllers/admin/SettingsController.php

```diff
@@ -135,6 +147,20 @@ public function postEdit()
             $setting->slack_endpoint = e(Input::get('slack_endpoint'));
             $setting->slack_channel = e(Input::get('slack_channel'));
             $setting->slack_botname = e(Input::get('slack_botname'));
+            $setting->ldap_enabled = e(Input::get('ldap_enabled', '0'));
+            $setting->ldap_server = e(Input::get('ldap_server'));
+            $setting->ldap_uname = e(Input::get('ldap_uname'));
+            $setting->ldap_pword = e(Input::get('ldap_pword'));
+            $setting->ldap_basedn = e(Input::get('ldap_basedn'));
+            $setting->ldap_filter = e(Input::get('ldap_filter'));
```

Comment:
Yeah I ran into that issue using the ampersand. I'll remove the e()'s here. So we would just add the e() to the view instead?